code/dynamic_programming/longest_bitonic_sequence/longest_bitonic_sequence.py

```diff
@@ -0,0 +1,59 @@
+# Part of Cosmos by OpenGenus Foundation
+
+import copy
+
+# returns the longest increasing sequence
+def longest_increasing_seq(numbers):
+    # longest increasing subsequence
+    lis = []
+    # base case
+    lis.append([numbers[0]])
+
+    # start filling using dp - forwards
+
+    for i in xrange(1, len(numbers)):
+        lis.append([])
+        for j in xrange(i):
+            if numbers[j] < numbers[i] and len(lis[j]) > len(lis[i]):
+                lis[i] = copy.copy(lis[j])
+        lis[i].append(numbers[i])
+    return lis
+
+# returns longest decreasing sequence
+def longest_decreasing_seq(numbers):
+    # longest decreasing subsequence
+    lds = []
+
+    # to make an n sized list
+    for _ in xrange(len(numbers)):
+        lds.append([])
+    lds = [[] for _ in xrange(len(numbers))]
```

Comment:
As you are already doing `lds = [[] for _ in xrange(len(numbers))]`, you don't need the code from line 25 to 29. It already does what these lines do.